### PR TITLE
Add taxonomic class to each ambassador

### PR DESCRIPTION
**Pull request by mforsb**
Companion pr for https://github.com/alveusgg/alveusgg/pull/288

### CODE REVIEW

**Comment by MattIPv4** [comment]
👀 youll want to include a version bump with this -- `npm version minor --no-git-tag-version` off the top of my head

File: src/ambassadors/classification.ts [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+  Diplopoda,
+  Insecta,
+  Malacostraca,
+}
+
+const classificationOrder = ["Mammals", "Birds", "Reptiles & Amphibians", "Invertebrates"] as const;
+
+export type Classification = (typeof classificationOrder)[number];
+
+export function getClassification(c: Class): Classification {
```
Comment (MattIPv4) [inline src/ambassadors/classification.ts]:
```suggestion
export const getClassification(c: Class): Classification => {
```
nitpick, I think existing code here prefers using arrow functions

File: src/ambassadors/classification.ts [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    case Class.Reptilia:
+    case Class.Amphibia: return "Reptiles & Amphibians";
+    case Class.Arachnida:
+    case Class.Diplopoda:
+    case Class.Insecta:
+    case Class.Malacostraca: return "Invertebrates";
+  }
+}
+
+export function sortAmbassadorClassification(a: Class, b: Class): number {
```
Comment (MattIPv4) [inline src/ambassadors/classification.ts]:
```suggestion
export const sortAmbassadorClassification(a: Class, b: Class): number => {
```
Ditto

File: src/ambassadors/classification.ts [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+  return classificationOrder.indexOf(getClassification(a)) - classificationOrder.indexOf(getClassification(b));
+}
+
```
Comment (MattIPv4) [inline src/ambassadors/classification.ts:37]: It looks like you've maybe got an extra newline at the end of the file?

**Comment by MattIPv4** [comment]
👀 Looks like you might need to run prettier locally?

**Comment by mforsb** [comment]
> eyes Looks like you might need to run prettier locally?

Oops! Done.

**Review by pjeweb — APPROVED** [review]

**Review by MattIPv4 — APPROVED** [review]

**Pull request merged**